src/Env/Defaults/Defaults.php

```diff
@@ -9,27 +9,27 @@
  * file that was distributed with this source code.
  */
 
-namespace Manala\Manalize\Env\Metadata;
+namespace Manala\Manalize\Env\Defaults;
 
 /**
- * A metadata bag containing all defaults for a given env type.
+ * A bag containing all defaults for a given env.
  *
  * @author Robin Chalas <robin.chalas@gmail.com>
  */
-final class MetadataBag
+final class Defaults
```

Comment:
Nevermind. I was actually confused beteween the actual metadata, defaults, and this class.
After pulling the code, this totally make sense... 👍 😆